### PR TITLE
Use phantomjs v2.1.3

### PR DESCRIPTION
**Pull request by fredj**
And remove the now unneeded Function.prototype.bind polyfill.

### CODE REVIEW

**Comment by fredj** [comment]
The `npm prune` in `travis.yml` is here to remove old packages from the travis cache

**Comment by fgravin** [comment]
+1 looks good thanks

File: Makefile [hunk 1]
```diff
@@ -44,7 +44,6 @@ EXAMPLE_HOSTED_REQUIREMENTS = .build/examples-hosted/lib/ngeo.js \
 	.build/examples-hosted/lib/watchwatchers.js \
 	.build/examples-hosted/lib/typeahead.bundle.min.js \
 	.build/examples-hosted/lib/proj4.js \
-	.build/examples-hosted/lib/Function.prototype.bind.js \
```
Comment (sbrunner) [inline Makefile]: Should be removed :-)

Comment (sbrunner) [inline Makefile]: allready don, :-)

File: .travis.yml [hunk 2]
```diff
@@ -6,6 +6,9 @@ cache:
   directories:
   - node_modules
 
+before_script:
+- npm prune
```
Comment (sbrunner) [inline .travis.yml]: Why this ?

Comment (fredj) [inline .travis.yml]: To cleanup the node_modules directory (from the travis cache)

Comment (fredj) [inline .travis.yml]: See https://docs.npmjs.com/cli/prune

**Comment by fredj** [comment]
Travis fails because of #572

File: .travis.yml (hunk 2 above)
Comment (sbrunner) [inline .travis.yml]: OK, thanks :-)

**Pull request closed without merging**